packages/web/components/complex/assets-page-v1.tsx

```diff
@@ -23,6 +25,23 @@ import { formatPretty } from "~/utils/formatter";
 
 const INIT_POOL_CARD_COUNT = 6;
 
+const TransactionsLink = () => {
+  const { t } = useTranslation();
+  return (
+    <div className="flex h-12 items-center justify-between rounded-3xl bg-osmoverse-850 px-4">
+      <div className="flex items-center gap-3 p-3">
+        <Icon id="history" />
+        <div className="subtitle1 text-osmoverse-100">
+          {t("transactions.title")}
+        </div>
+      </div>
+      <Button variant="ghost" asChild className="text-wosmongton-200" size="md">
```

Comment:
This is a button element just used for styling? Are there any accessibility concerns there? (But IDK what `asChild` is doing, to be fair)